### DISCUSSION TITLE
Indent level as text object

### DISCUSSION THREAD

**Discussion by Orenjonas** [lukas-reineke/indent-blankline.nvim#581] [Ideas]
Hi!
First of all, thanks for an amazing plugin. I have been looking for a way to get indent level as text objects for python and I was wondering if you have you considered using the indent logic behind this plugin for that purpose?

**Comment by lukas-reineke** [comment]
No this is out of scope for this plugin.
But there are plugins that do this, for example https://github.com/michaeljsmith/vim-indent-object

**Reply by Orenjonas** [reply]
Ah, thanks for the tip! Thats sloppy research on my part.
